recipes/ncbi-cxx-toolkit-public/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "26.0.1":
+    folder: "26.0"
```

Comment:
for maintenance perspective: do you expect deep changes between minor versions? If not, it's better to just put this recipe under `all` folder.

Reply:
I do not expect deep changes between minor versions. that is why all 26.0.x (if any) will use ""26.0" subfolder. I expect deep changes between major versions. that is why all 27.x versions will use "27.0" subfolder. there is no point in having "all"

Reply:
So folder name should be 26.x

Reply:
sure